turbopack/crates/turbopack-ecmascript-runtime/js/src/browser/runtime/base/runtime-base.ts

```diff
@@ -74,7 +74,10 @@ type SourceInfo =
 
 interface RuntimeBackend {
   registerChunk: (chunkPath: ChunkPath, params?: RuntimeParams) => void
-  loadChunk: (chunkUrl: ChunkUrl, source: SourceInfo) => Promise<void>
+  /**
+   * Returns the same Promise for the same chunk URL.
+   */
+  loadChunkCached: (chunkUrl: ChunkUrl, source: SourceInfo) => Promise<void>
```

Comment:
Just added as a drive-by since I got confused for a second by its usage in `loadChunkByUrl`